GetStream/stream-chat-android
# [3875] Fix scroll to bottom on new message in thread

### 🎯 Goal

Closes #3875  

### 🛠 Implementation details

Add new message state check to thread messages.

### 🎨 UI Changes
<table>
<thead>
<tr>
<th>Before</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<video src="https://user-images.githubusercontent.com/38032787/179959644-ecf4a425-1dcf-44f6-8721-90b0661f090c.mp4" controls="controls" muted="muted" />
</td>
<td>
<video src="https://user-images.githubusercontent.com/38032787/179959631-4cc1a940-78ec-4c5c-a32c-646fdabbdf1b.mp4" controls="controls" muted="muted" />
</td>
</tr>
</tbody>
</table>

### 🧪 Testing

1. Build on develop and send messages to a thread
2. Scroll to new message should not be working
3. Build this branch and send messages to a thread
4, Scroll to new message should be working

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

![giphy (1)](https://user-images.githubusercontent.com/38032787/179960472-e910dc32-ca98-46b1-8821-f8e71a90c067.gif)
